### PR TITLE
Use custom dmoj select2 theme for dark mode support

### PR DESCRIPTION
**Pull request by Ninjaclasher**
Part of #2035. This PR will make the problem submit language select2 look bad due to a lot of custom CSS, but it will be fixed in a followup PR as this one is getting too large.

After:
image
image

image
image

The admin site only uses the light mode version:
image

### CODE REVIEW

**Comment by Ninjaclasher** [comment]
Changed so the search dropdown on light theme has a white background, so now it looks like this:
image

File: judge/template_context.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def get_resource(request):
         'FONTAWESOME_CSS': settings.FONTAWESOME_CSS,
         'DMOJ_SCHEME': scheme,
         'DMOJ_CANONICAL': settings.DMOJ_CANONICAL,
+        'SELECT2_THEME': settings.DMOJ_SELECT2_THEME,
```
Comment (Xyene) [inline judge/template_context.py:33]: For consistency with the other settings, this should be

```suggestion
        'DMOJ_SELECT2_THEME': settings.DMOJ_SELECT2_THEME,
```

Comment (Ninjaclasher) [reply judge/template_context.py]: Done

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**